cmd/root.go

```diff
@@ -111,6 +117,60 @@ func args() []string {
 	return args
 }
 
+func bugReport(ctx context.Context) error {
+	var containerDaemonPaths = []string{
+		"/var/run/docker.sock",
+		"/var/run/podman/podman.sock",
+		"$HOME/.colima/docker.sock",
+	}
+
+	report := fmt.Sprintf("act version:\t%s\n", Version)
+	report += fmt.Sprintf("GOOS:\t\t%s\n", runtime.GOOS)
+	report += fmt.Sprintf("GOARCH:\t\t%s\n", runtime.GOARCH)
+	report += fmt.Sprintf("NumCPU:\t\t%d\n", runtime.NumCPU())
+
+	if os.Getenv("DOCKER_HOST") == "" {
+		report += fmt.Sprintln("Sockets found:")
+		for _, p := range containerDaemonPaths {
+			if strings.HasPrefix(p, "$HOME") {
+				p = strings.Replace(p, "$HOME", os.Getenv("HOME"), 1)
+			}
+			if _, err := os.Stat(p); err != nil {
+				continue
+			} else {
+				report += fmt.Sprintf("\t%s\n", p)
+			}
+		}
+	}
+
+	info, err := container.GetHostInfo(ctx)
+	if err != nil {
+		return err
+	}
+
+	report += fmt.Sprintln("Docker Engine:")
+
+	report += fmt.Sprintf("\tEngine version:\t%s\n", info.ServerVersion)
+	report += fmt.Sprintf("\tCgroup version:\t%s\n", info.CgroupVersion)
+	report += fmt.Sprintf("\tEngine runtime:\t%s\n", info.DefaultRuntime)
+
+	report += fmt.Sprintf("\tOS:\t\t%s\n", info.OperatingSystem)
+	report += fmt.Sprintf("\tOS type:\t%s\n", info.OSType)
+	report += fmt.Sprintf("\tOS version:\t%s\n", info.OSVersion)
+	report += fmt.Sprintf("\tOS arch:\t%s\n", info.Architecture)
+	report += fmt.Sprintf("\tOS kernel:\t%s\n", info.KernelVersion)
+	report += fmt.Sprintf("\tOS CPU:\t\t%d\n", info.NCPU)
+	report += fmt.Sprintf("\tOS memory:\t%d\n", info.MemTotal)
```

Comment:
Just a small nitpick, but can we format the bytes human readable?

Reply:
sure, I have yet to still get rid of `\t`s and `\n`s 😸

Reply:
done